src/scripts/cronjobs/firstDataBrokerRemovalFixed.tsx

```diff
@@ -145,3 +146,13 @@ async function sendFirstDataBrokerRemovalFixedActivityEmail(
     ),
   );
 }
+
+function isFulfilledResult(
```

Comment:
I know this was just moved and not new, just curious why this is needed? Does it do the same thing as [Promises.allSettled()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled)?